components/reddit/package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "reddit",
```

```suggestion
  "name": "@pipedream/reddit",
```
I guess this would be `@pipedream/reddit`

---

Can I commit this if I am not wrong?